packages/webpack/lib/plugins/log.js

```diff
@@ -70,6 +70,15 @@ exports.LoggerPlugin = class LoggerPlugin {
           this.logger.info(formatWarning(name, duration, assets, isRebuild));
         } else {
           this.logger.info(formatSuccess(name, duration, assets, isRebuild));
+          const assetsOverSizeLimit = assets.some(
```

Comment:
I think we should print this message even if we have other warnings...
And without the `-v` flag the hint is not visible, but I guess that is intentional.

Reply:
@jhiode we are considering raising the log level in non-production environments. That would make these hints visible in development mode.

Reply:
@jhiode I moved the warning / hint one level up, so it will be printed even if the build has other warnings.